applications/GeoMechanicsApplication/tests/fixed_spatial_variation/ProjectParameters.json

```diff
@@ -0,0 +1,168 @@
+{
+    "problem_data": {
+        "problem_name":         "simple_mesh",
+        "start_time":           0.0,
+        "end_time":             1.0,
+        "echo_level":           1,
+        "parallel_type":        "OpenMP",
+        "number_of_threads":    1
+    },
+    "solver_settings": {
+        "solver_type":                        "geomechanics_U_Pw_solver",
+        "model_part_name":                    "PorousDomain",
+        "domain_size":                        2,
+        "model_import_settings":              {
+            "input_type":       "mdpa",
+            "input_filename":   "simple_mesh"
+        },
+        "material_import_settings":              {
+            "materials_filename":       "MaterialParameters.json"
+        },
+        "time_stepping":              {
+            "time_step":                1.0,
+            "max_delta_time_factor":    1000
+        },
+        "buffer_size":                        2,
+        "echo_level":                         1,
+        "clear_storage":                      false,
+        "compute_reactions":                  false,
+        "move_mesh_flag":                     false,
+        "reform_dofs_at_each_step":           false,
+        "nodal_smoothing":                    false,
+        "block_builder":                      true,
+        "solution_type":                      "Quasi-static",
+        "scheme_type":                        "Backward_Euler",
+        "reset_displacements":                true,
+        "newmark_beta":                       0.25,
+        "newmark_gamma":                      0.5,
+        "newmark_theta":                      0.5,
+        "rayleigh_m":                         0.0,
+        "rayleigh_k":                         0.0,
+        "strategy_type":                      "newton_raphson",
+        "convergence_criterion":              "residual_criterion",
+        "displacement_relative_tolerance":    1.0E-4,
+        "displacement_absolute_tolerance":    1.0E-9,
+        "residual_relative_tolerance":        1.0E-4,
+        "residual_absolute_tolerance":        1.0E-9,
+        "water_pressure_relative_tolerance":  1.0E-4,
+        "water_pressure_absolute_tolerance":  1.0E-9,
```

Comment:
Maybe leave out the displacement and water_pressure criteria parameters. They are not used.

Reply:
Removed (along with a few others, it can probably be even leaner, but let's have a broader discussion first, also about defaults)